awsiot/iotshadow.py

```diff
@@ -174,7 +175,9 @@ def publish_update_shadow(self, request, qos):
             payload=request.to_payload())
 
     def subscribe_to_delete_named_shadow_accepted(self, request, qos, callback):
-        # type: (DeleteNamedShadowSubscriptionRequest, int, typing.Callable[[DeleteShadowResponse], None]) -> typing.Tuple[concurrent.futures.Future, str]
+        # type: (DeleteNamedShadowSubscriptionRequest, int,
+        # typing.Callable[[DeleteShadowResponse], None]) ->
+        # typing.Tuple[concurrent.futures.Future, str]
```

Comment:
undo these whitespace changes if you can

if you can't figure out why this whitespace change happened, please verify that this is valid syntax for python type hints